general/db/guide-gigaspaces.asciidoc

```diff
@@ -0,0 +1,15 @@
+:toc: macro
+toc::[]
+
+= GigaSpaces XAP
+
+This section is the place to share experience for those who use https://www.gigaspaces.com/products/xap/[GigaSpaces XAP] as link:guide-database.asciidoc#nosql[NoSQL database].
+
+== Attention
+NOTE: In devonfw we do not properly support this. You are free to use this data storage but do not expect advanced integration, CobiGen templates, etc.

```

Comment:
> NOTE: In devonfw we do not properly support this.
what does that mean? Could we be a little be more precise? Does this mean, "if you want gigaspaces, you cannot use devonfw"? Could we be a little be more solution focused rather than what you not get with gigaspaces?
Could we provide we here for example "what we did to integrate?" possibly a katacoda link? 
If the user reads this here, he will stop reading and will not have a look at the below for sure.